packages/fxa-settings/.storybook/design-guide/Header.tsx

```diff
@@ -5,15 +5,15 @@ const Header = () => {
   return (
     <div className="bg-grey-50 flex items-center justify-between sticky z-20 top-0 pt-4 pb-3 border-b border-grey-400/25 px-2 -mt-4 -ml-12 -mr-12">
       <div className="flex items-center">
-        <img src="logo.svg" className="mr-3" />
+        <img src="logo.svg" className="mr-3" alt="firefox-account" />
```

Comment:
I doubt we'll get many screen reader users on our design guide and I think this would be read the same anyways but moreso for future reference, this should probably just be `firefox account` or even `firefox account logo` instead of `firefox-account`. Not worth another CI run though if this is the only thing that'd need a tweak.